.github/workflows/dapr-test.yml

```diff
@@ -30,7 +30,11 @@ on:
   # Dispatch on external events
   repository_dispatch:
     types: [e2e-test]
-
+  pull_request:
```

Comment:
Please don't add this. For PRs, we will need to use `/ok-to-test`.

During development on your own, you will probably need to create your own fork. Feel free to ping me offline and I'll explain how!

Reply:
Yeah I only added this to test it.